### PR TITLE
Update daikin doc

### PR DESCRIPTION
**Pull request by mlemainque**
## Proposed change

Update the documentation for https://github.com/home-assistant/core/pull/78231

## Type of change

- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/78231
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes https://github.com/home-assistant/core/issues/72545

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/daikin.markdown [hunk 1]
```diff
@@ -95,16 +95,20 @@ _When "Holiday mode" is enabled, the following action take place:_
...
 
 - Inside temperature
-- Outside temperature
 - Inside humidity
-- Estimated power consumption
 - Hourly energy consumption in cool mode
 - Hourly energy consumption in heat mode
+- Today's total energy consumption (updated hourly, resets at 00:00)
+
+The integration displays the following parameters for the outdoor compressor:
```
Comment (MartinHjelmare) [inline source/_integrations/daikin.markdown:106]: The outdoor measurements were removed from the PR, right?

**Comment by frenck** [comment]
@mlemainque Friendly ping, there is a merge conflict and a review comment open for this PR. Could you take a look?

File: source/_integrations/daikin.markdown (hunk 1 above)
Comment (mlemainque) [reply source/_integrations/daikin.markdown:106]: The "outdoor" device has been removed but not the sensors, and IMO it's still worth distinguishing indoor and outdoor sensors in the doc

File: source/_integrations/daikin.markdown [hunk 2]
```diff
@@ -117,11 +121,12 @@ The `daikin` sensor platform integrates Daikin air conditioning systems into Hom
...
 <div class='note'>
 
-- The 'Today's total energy consumption' and 'Estimated power consumption' sensor is updated every time 100 Wh are consumed by all different operating modes summed together. These values are the values for the whole installation, not per device (unless, you have only one device).
-- The 'Estimated power consumption' sensor is derived from the energy consumption and not provided by the AC directly.
+- The 'Outdoor compressor Energy consumption' and 'Outdoor compressor Estimated power consumption' sensors are updated every time 100 Wh are consumed by all different operating modes summed together.
+- The 'Outdoor compressor Estimated power consumption' sensor is derived from the above energy consumption and not provided by the AC directly.
 - The 'cool/heat' energy sensors are updated hourly with the previous hour energy consumption
   of a given mode and a given AC.
 - The 'cool' mode also includes the 'fan' and 'dehumidifier' modes' power consumption.
+- Would you have multiple indoor devices, the 'Outdoor compressor' sensors will be created multiple times but will all report the same values. You should disable all but one.
```
Comment (MartinHjelmare) [inline source/_integrations/daikin.markdown]:
```suggestion
- If you have multiple indoor devices, the 'Outdoor compressor' sensors will be created multiple times but will all report the same values. You can disable all but one.
```

File: source/_integrations/daikin.markdown (hunk 1 above)
Comment (MartinHjelmare) [reply source/_integrations/daikin.markdown:106]: Ok.

**Review by frenck — APPROVED** [review]
Thanks, @mlemainque 👍

**Pull request merged**